src/index.js

```diff
@@ -179,14 +185,17 @@ function traverse (node, context) {
 }
 
 function renderToStream (node) {
+  // Reset REACT_ID so each call starts from 1.
```

Comment:
Hmm, we might want to pass the `reactId` to `traverse` and have it increment it every time it recurses instead of relying on a global.